src/uct/api/v2/uct_v2.h

```diff
@@ -604,6 +617,29 @@ ucs_status_t uct_ep_query(uct_ep_h ep, uct_ep_attr_t *ep_attr);
 int uct_iface_is_reachable_v2(uct_iface_h iface,
                               const uct_iface_is_reachable_params_t *params);
 
+
+/**
+ * @ingroup UCT_RESOURCE
+ * @brief Connect endpoint to a remote endpoint.
+ *
+ * requires @ref UCT_IFACE_FLAG_CONNECT_TO_EP capability.
+ *
+ * @param [in] ep           Endpoint to connect.
+ * @param [in] device_addr  Remote device address.
+ * @param [in] iface_addr   Remote interface address.
+ * @param [in] ep_addr      Remote endpoint address.
+ * @param [in] params       Parameters as defined in @ref
+ *                          uct_ep_connect_to_ep_params_t.
+ *
+ * @return UCS_OK           Operation has been initiated successfully.
+ *         Other error codes as defined by @ref ucs_status_t.
```

Comment:
minor: formatting is not aligned with the previous line

Reply:
will update on squash